packages/manager/modules/netapp/src/dashboard/volumes/routing.js

```diff
@@ -0,0 +1,82 @@
+import Share from './Share.class';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('netapp.dashboard.volumes', {
+    url: '/volumes',
+    views: {
+      '@netapp.dashboard': {
+        component: 'ovhManagerNetAppVolumes',
+      },
+    },
+    resolve: {
+      detailedVolumes: /* @ngInject */ ($http, $q, serviceName, volumes) =>
+        $q.all(
+          volumes.map((volume) =>
+            $q
+              .all({
```

Comment:
Wouldn't it be better to use a `row-loader` instead ?

Reply:
Done, thanks